### PR TITLE
1 add jupyter notebook support

### LINKED ISSUE #1: Add jupyter notebook support
Add mkdocs add on to support rendering of jupyter notebooks.

Link to resource: <https://github.com/danielfrg/mkdocs-jupyter>

### PR DESCRIPTION
**Pull request by iYadavVaibhav**
Adds support to render jupyter notebook.

## Steps to Run:

- Clone and install requirements
- Run with `mkdocs serve`

## Tests:

- Verify you can view a rendered `*.ipynb` on site.

## Known Issues:

- Render Latex and Plotly charts (at times)

### CODE REVIEW

File: requirements.txt
Comment (CodeoMascot) [inline requirements.txt:1]: please add missing requirement for `mdx_truly_sane_lists`

**Review by CodeoMascot — CHANGES_REQUESTED** [review]
Error when running, there is a missing extension.

```sh
ERROR    -  Config value 'markdown_extensions': Failed to load extension 'mdx_truly_sane_lists'.
            ModuleNotFoundError: No module named 'mdx_truly_sane_lists'
Aborted with 1 Configuration Errors!
```

**Review by CodeoMascot — CHANGES_REQUESTED** [review]
Please add to requirements

File: requirements.txt
Comment (iYadavVaibhav) [reply requirements.txt:1]: Added, thanks

**Review by CodeoMascot — APPROVED** [review]
Working as expected

**Review by CodeoMascot — APPROVED** [review]

**Pull request merged**